google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -74,9 +74,19 @@ appengine.flex.configtype.custom.label=Custom
 appengine.flex.user.specified.artifact.label=Select a deployment archive:
 appengine.flex.dockerfile.location.label=Dockerfile path\:
 appengine.flex.version.placeholder.text=auto generated
-appengine.flex.config.destination.folder.label=Choose destination folder\:
+appengine.flex.facet.config.title=App Engine flexible configuration
+appengine.flex.facet.config.appyaml.error.before.text=The specified app.yaml is invalid or does not exist. Browse to an app.yaml file, or 
+appengine.flex.facet.config.dockerfile.directory.error.before.text=The directory does not contain a Dockerfile. Browse to a directory with a Dockerfile, or 
+appengine.flex.facet.config.runtime.explanation.label=if you want to provide a Dockerfile, change the runtime to 'custom' in app.yaml
+appengine.flex.facet.config.appyaml.generate.link.text=generate a new app.yaml ...
```

Comment:
@patflynn there are a lot of cases all over the source where "app.yaml" is used. Rather then polluting this PR I suggest we do this separately. I can file an issue.